dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java

```diff
@@ -95,27 +95,39 @@ public Result createProcessDefinition(@ApiIgnore @RequestAttribute(value = Const
     }
 
     /**
-     * copy process definition
+     * copy or move process definition
      *
      * @param loginUser   login user
      * @param projectName project name
-     * @param processId   process definition id
+     * @param isCopy  isCopy
+     * @param processDefinitionIds   process definition ids
+     * @param targetProjectId target project id
      * @return copy result code
      */
-    @ApiOperation(value = "copyProcessDefinition", notes= "COPY_PROCESS_DEFINITION_NOTES")
+    @ApiOperation(value = "copyOrMoveProcessDefinition", notes= "COPY_OR_MOVE_PROCESS_DEFINITION_NOTES")
     @ApiImplicitParams({
-            @ApiImplicitParam(name = "processId", value = "PROCESS_DEFINITION_ID", required = true, dataType = "Int", example = "100")
+            @ApiImplicitParam(name = "processDefinitionIds", value = "PROCESS_DEFINITION_IDS", required = true, dataType = "String", example = "3,4"),
+            @ApiImplicitParam(name = "targetProjectId", value = "TARGET_PROJECT_ID", required = true, type = "Integer"),
+            @ApiImplicitParam(name = "isCopy", value = "IS_COPY", required = true, type = "boolean")
     })
-    @PostMapping(value = "/copy")
+    @PostMapping(value = "/copy-or-move")
```

Comment:
Hi,
Will it be better to split copy and move to two api for single responsibility?